src/utils/general-helpers.js

```diff
@@ -57,12 +57,11 @@ function clearLocalRequireCache(themePath) {
  * @param {Object} options
  * @param {Number} exitCode
  */
-function exitHandler(options, exitCode) {
+function exitHandler(options) {
   if (options.cleanup) {
     rmdirRecursiveSync('.debug');
     console.log('\n\nBiee 🐨✌️\n');
   }
-  if (exitCode !== 0) console.log(exitCode);
```

Comment:
I exit now with code 1 when we exceed max retries. Displaying the exit code is non ideal here so I had to remove it.

Reply:
Yup makes sense